.env

```diff
@@ -1 +1,23 @@
 # This file is here for dynaconf to set this as root level directory
+
+# Set Vault Enable to true to access secrets from vault
+VAULT_ENABLED_FOR_DYNACONF=false
+VAULT_URL_FOR_DYNACONF=http://localhost:8200
+# Specify the secrets engine for kv, default is 1
+VAULT_KV_VERSION_FOR_DYNACONF=2
+# Specify certs to access secrets on https vault
+VAULT_VERIFY_FOR_DYNACONF=false
+# Authenticate with token https://www.vaultproject.io/docs/auth/token
+# VAULT_TOKEN_FOR_DYNACONF=myroot
+# Authenticate with AppRole https://www.vaultproject.io/docs/auth/approle
+# VAULT_ROLE_ID_FOR_DYNACONF=
+# VAULT_SECRET_ID_FOR_DYNACONF=
+# Authenticate with AWS IAM https://www.vaultproject.io/docs/auth/aws
+# The IAM Credentials can be retrieved from the standard providers: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+# VAULT_AUTH_WITH_IAM_FOR_DYNACONF=False
+# VAULT_AUTH_ROLE_FOR_DYNACONF=vault-role
+# Authenticate with root token
+# VAULT_ROOT_TOKEN_FOR_DYNACONF=root-token
+# Path of secrets
+VAULT_MOUNT_POINT_FOR_DYNACONF=apps
+VAULT_PATH_FOR_DYNACONF=mna-satqe-hashicorp/robottelo
```

Comment:
I think this should be far more generic for this field.

Reply:
Didnt get the comment :) Can you elaborate on this ?

Reply:
This file in our upstream open-source repo should not contain the vault path that we use internally.